closed_issues.rb

```diff
@@ -1,12 +1,14 @@
-MILESTONE = 36
-ACCESS_TOKEN = "your github access token"
+MILESTONE = 47
+# subtract 6 from actual Sprint milestone number for the ManageIQ/manageiq repo
```

Comment:
The milestone can be found by either its ID or name.  The name text would have to match exactly, but if it's consistent, maybe that would be an easier thing to search for.  @mfeifer What do you think?